test/controllers/admin/users_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class Admin::UsersControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Best. Comment. EVAR

Reply:
Fun thing is this project should have rspec but the generators still build minitest tests. Oh and I don't have any tests. Yet.